### PR TITLE
[UI] Balance fix + bubble-help + usability improvements

### PR DESCRIPTION
**Pull request by Mrs-X**
Includes:
- fix of wrong/inconsistent display of balances
- tons of bubble-help to explain what's displayed to the end user
- include unconfirmed/immature/mature zPIV balances on the Overview-tab
- distinguish between unconfirmed and immature zPIV on Privacy-tab
- general usability improvements

Visuals:
Overview-Tab:
overview-tab

Privacy-Tab:
privacy-tab

Please check spelling, grammar and lyrical virtue and report back...

### CODE REVIEW

**Comment by presstab** [comment]
utACK looks straight forward.

**Comment by jonspock** [comment]
On Bubble help, rather than saying "If Automint" is enabled, can you determine that and change the message based on the value of automint. Plus give instructions on how to enable or disable

File: src/qt/forms/overviewpage.ui [hunk 1]
```diff
@@ -923,8 +907,11 @@
                  <bold>true</bold>
                 </font>
                </property>
+               <property name="toolTip">
+                <string>Combined Balances (including unconfirmed and immature coins)</string>
+               </property>
                <property name="text">
-                <string>Combined Balances (including immature coins)</string>
+                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Combined Balances&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```
Comment (Fuzzbawls) [inline src/qt/forms/overviewpage.ui]: Is the inline html in these string elements (here and elsewhere) really necessary? The raw contents get sent off to transifex for translation and having the html tags embedded in the string can and most likely will cause translator confusion and/or error.

**Comment by jonspock** [comment]
Available for zPIV is the wrong title in more than 1 place as "immature" and "unconfirmed" coins are not available. In the case of PIV on main screen. "Available" means available for immediate spending, it seems to mean something else for zPIV

**Comment by jonspock** [comment]
NACK until mentioned issues are addressed

**Comment by s3v3nh4cks** [comment]
While you are adjusting the Overview Tab, can you add the row for collateral/locked coins so the percentages look correct .. :)

File: src/qt/forms/overviewpage.ui (hunk 1 above)
Comment (Mrs-X) [reply src/qt/forms/overviewpage.ui]: Translatability over usability...that's a new one :-)

IMO having **bold** text elements helps to read the online help text, but on the other side I also see the problems with translation (I have translated quite some texts with HTML in the past, though).
Will change it.

**Comment by Mrs-X** [comment]
@s3v3nh4cks I *knew* this would be a kind of *Hydra*  PR...

**Comment by Mrs-X** [comment]
So, finally all change requests integrated and some more help/usability improvements.

@Fuzzbawls  HTML code removed.

@jonspock Improved AutoMint bubble-help:
automint-enabled
automint-disabled

@s3v3nh4cks 'Locked' and 'Unlocked' coins added:
overview


Additional (non-visible) changes:
- Privacy tab: Available for minting excludes locked coins now
- Overview tab: Percentage is now displayed based on avilable unlocked(!) coins.

Intentionally not squashed because I'd like to have the second commit as reference for the Locked/Unlocked coins refresh hack.

**Comment by jonspock** [comment]
Changing to ACK based on screenshots

**Pull request merged**